amida-tech/amida-messaging-microservice
# Bug/SER-130 Fix type checking assertions in unit tests

#### What's this PR do?
Fix type checks assertions in unit tests

Fix linter errors that were preventing me from committing

#### Related JIRA tickets:
[SER-130](https://jira.amida-tech.com/browse/SER-130)

#### How should this be manually tested?
Check that all chai assertions end in a function call and that all tests pass. Change types in the assertions and check that the tests fail.

#### Any background context you want to provide?
#### Screenshots (if appropriate):
